exporter/metric/metric_test.go

```diff
@@ -75,9 +75,9 @@ func TestRecordToMpb(t *testing.T) {
 
 	desc := apimetric.NewDescriptor("testing", apimetric.ValueRecorderKind, apimetric.Float64NumberKind)
 
-	lvagg := lastvalue.New()
+	lvagg := &lastvalue.Aggregator{}
```

Comment:
It might be better to use a constructor function here
```suggestion
	lvagg := &lastvalue.New(1)[0]
```